www/js/calendar.js

```diff
@@ -36,10 +36,6 @@ function initCalendar(page) {
 
       // Initialize "today click" listener
       toolbar.find('.right').on('click', todayClick);
-
-      // Display today's event content
-      var todayContainer = page.find('.picker-calendar-day-today');
-      displayDayContent(p.params.event, todayContainer);
```

Comment:
displayDayContent() already runs after loadEvents() -> updateDayContent() -> displayDayContent(), so we don't need to run it here